### ISSUE TITLE
[VS Classic] Implement status bar

### ISSUE THREAD

**Issue by olegklimov** [smallcloudai/refact-vs-classic#8]
It needs to show:
- "|{ Refact" when idle
- rotating icon of some sort when it's thinking about the new completion
- contrast background and "unplugged" icon when error, tooltip should show what the error was

**Comment by digital-phoenix** [comment]
@olegklimov or did you just mean that the unplugged icon should visually stand out

**Comment by olegklimov** [comment]
@digital-phoenix unplugged is just this:

<img width="418" alt="image" src="https://github.com/smallcloudai/refact-vs-classic/assets/18546213/e0f46144-70a6-4684-99d6-6f8f12815d84">

And this:

<img width="364" alt="image" src="https://github.com/smallcloudai/refact-vs-classic/assets/18546213/efbd0349-505d-4051-a879-f916bd6a9f69">

Give the LSP server a wrong address or something, you'll see an error. Or turn off Wi-Fi on your laptop.

**Comment by digital-phoenix** [comment]
@olegklimov do you have an image I could use for "|{ Refact" or the disconnected plug icon?

**Comment by olegklimov** [comment]
Almost! We need to give you @digital-phoenix a rotating icon. And a disconnected plug.

**Comment by hazratisulton** [comment]
no status popup, no icon changes when hovering;    
no error message if you delete api key, close settings, open settings and add new incorrect key - still working;    
no error message when change correct url to incorrect; 
if no API key there is message "500 Internal Server error, no caps available" instead of "Missing authorization header";   
incorrect message "The API key is not recognized" when API key is incorrect; 
need to reload IDE when added correct API key;

**Comment by olegklimov** [comment]
<img width="273" alt="image" src="https://github.com/smallcloudai/refact-vs-classic/assets/18546213/db12db74-d171-40e0-b7ee-0bbdea99159c">

My list of unfinished things:
- Popup that says which model was last used
- Error message in the same popup, error icon, maybe error background in status bar
- Errors that should make it into the popup (and icon): any LSP errors (give it a misspelled model), connection problems (turn wifi off), bad API key (especially important for the message to be intelligible).
- If the API key is empty, popup should give a link to README on github to the #installation header, so people know what they need to do to make it work

**Comment by olegklimov** [comment]
No, it's too intrusive. Background and icon change are good ideas. But not a popup that might be annoying.

**Comment by guptadeepak8** [comment]
@olegklimov i think the rotating icon is working what else need to be done
